pyproject.toml

```diff
@@ -15,9 +15,10 @@ readme = {file = "README.md", content-type = "text/markdown"}
 requires-python = ">=3.8"
 
 [project.urls]
+homepage = "https://nbara.github.io/python-meegkit"
```

Comment:
conda-forge complained that there was no `homepage` so it must be pretty standard to include it